### DISCUSSION TITLE
Docker LXC (Portainer)

### DISCUSSION THREAD

**Discussion by Akenjeru** [tteck/Proxmox#2936] [Q&A] [answered]
How to update portainer lxc to 2.19.5 or 2.20?

**Comment by bramsuurdje** [comment, marked as answer]
inside of the docker container. you can run the following commands to update portainer to the latest version:

First, stop the container
`docker stop portainer`

Then, remove the old container
`docker rm portainer`

Pull the latest image:
`docker pull portainer/portainer-ce:latest`

Finally, re-deploy the container
`docker run -d -p 8000:8000 -p 9443:9443 --name=portainer --restart=always -v /var/run/docker.sock:/var/run/docker.sock -v portainer_data:/data portainer/portainer-ce:latest`
